sdk/core/Azure.Core/src/Shared/OperationInternalBase.cs

```diff
@@ -105,7 +119,7 @@ protected OperationInternalBase(ClientDiagnostics clientDiagnostics, Response ra
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>The last HTTP response received from the server, including the final result of the long-running operation.</returns>
         /// <exception cref="RequestFailedException">Thrown if there's been any issues during the connection, or if the operation has completed with failures.</exception>
-        public virtual async ValueTask<Response> WaitForCompletionResponseAsync(CancellationToken cancellationToken)
+        public async ValueTask<Response> WaitForCompletionResponseAsync(CancellationToken cancellationToken)
```

Comment:
Should these remain virtual for mocking? This would also be a breaking binary change: can't `callvirt` a non-virtual method.

Reply:
This type is internal